pkg/annotations/annotations.go

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package annotations
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// ExternalResourceGCAnnotation is the name of an annotation that indicates if
+	// external resources should be garbage collected for the cluster.
+	ExternalResourceGCAnnotation = "aws.cluster.x-k8s.io/external-resource-gc"
+)
```

Comment:
I think this the annotation must be part of our public API. For example, [this annotation](https://github.com/kubernetes-sigs/cluster-api/blob/0571cc51ca53bb568dd7b62d3c66b62498550c16/api/v1beta1/common_types.go#L44) in the Cluster API core API.

We already have an experimental part of the API, where you have added the new finalizers. Should we move the annotation there?

Reply:
I agree, this has been moved within `exp/api`

Reply:
Thanks